app/src/main/java/org/mozilla/fenix/settings/Extensions.kt

```diff
@@ -14,6 +17,25 @@ import mozilla.components.support.ktx.android.view.putCompoundDrawablesRelative
 import org.mozilla.fenix.R
 import org.mozilla.fenix.theme.ThemeManager
 
+/**
+ * Checks for availability of network.
+ *
+ * For devices above [Build.VERSION_CODES.M] it even checks if there's internet flowing through it or not.
+ * */
+fun ConnectivityManager.isOnline(): Boolean {
```

Comment:
Since it's used outside settings, could you move this to org/mozilla/fenix/ext/ConnectivityManager.kt?

Reply:
Yes. Makes sense. Will do.